src/playground/public/assets/js/demosData.ts

```diff
@@ -108,4 +108,44 @@ ask {
   }
 }`,
   },
+
+  demo51: {
+    title: 'Indian Covid API query with map',
+    code: `ask {
+      fetch('https://api.covid19india.org/data.json')['cases_time_series']
+      :map(fun(dataSet) {
+                             return {
+                                data: dataSet['date'],
+                                dailyconfirmed: dataSet['dailyconfirmed'],
+                                dailydeceased: dataSet['dailydeceased'],
+                                dailyrecovered: dataSet['dailyrecovered']
+                              }
+                            })
```

Comment:
let's fix the indentation

Reply:
Also `return is not needed here

Reply:
This mapping is essentially picking keys from objects. Shall we change key names to make this a little more exciting?